groups/sig-node/groups.yaml

```diff
@@ -146,6 +146,20 @@ groups:
       - pjbgf@linux.com
       - saschagrunert@gmail.com
 
+  - email-id: k8s-infra-staging-dra-driver-cpu@kubernetes.io
+    name: k8s-infra-staging-dra-driver-cpu
+    description: |-
+      ACL for pushing dra-driver-cpu artifacts
+    settings:
+      ReconcileMembers: "true"
+    members:
```

Comment:
this list should also typically match the OWNERS of the source repo but /shrug

we can iterate, just know that you're giving [staging only] push access to people that do not have merge access in the source repo ... if these people should have push access they probably should also have /approve access in that repo

Reply:
But the SIG can choose to manage this how they wish.